ospo-book/chapters/00-chapter.md

```diff
@@ -0,0 +1,27 @@
+# What is this book About?
+
+This book provides a guide to Open Source Programs Offices (OSPOs), to better understand their role in organizations, and the best practices for 
+establishing and managing an OSPO. The book covers a wide range of topics, including the reasons why organizations may need (or not) an OSPO, the 
+role of OSPOs in different types of organizations (public, private, small, medium, large), the challenges, antipatterns associated with OSPOs, and the 
+tools and processes required to establish and manage an effective OSPO. The book also does a brief introduction to assess an organization's readiness 
+for OSPO. As well as how to’s recommendations: from how to develop open source policies to how to engage with external open source communities.
+
+# What’s not in this book
+
+This book is not intended to be a guide to open source software development, nor does it cover the technical aspects of open source software development 
+in depth. Instead, the book focuses on the organizational and management aspects of open source dynamics within organizations by establishing OSPO roles. 
+
+The book neither provide detailed technical guidance on specific open source software projects, nor does it provide guidance on how to write or contribute 
+code to open source projects as individuals.
+
+# Who should read this book?
+
+This book is intended for any person that wants to better understand the role of OSPOs within an organization. The book may be of interest to any role 
+in an organization that will be or is currently involved in open source at some level and has direct relationship with OSPO roles (or is in one) or is 
+likely to have a close relationship with an OSPO in the future.
+
+* Executives, policy-makers and decision-makers who are responsible for establishing, overseeing, funding or sponsoring an OSPO within an organization.
+* Open source program managers and leaders who are responsible for coordinating and managing an organization's open source activities and engagement with external open source communities.
```

Comment:
External could be misinterpreted in a way that projects not started by the given organisation. 
```suggestion
* Open source program managers and leaders who are responsible for coordinating and managing an organization's open source activities and engagement with open source communities.
```